app/grandchallenge/reader_studies/models.py

```diff
@@ -0,0 +1,310 @@
+from collections import Counter
+
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import Group
+from django.contrib.postgres.fields import JSONField
+from django.db import models
+from django_extensions.db.models import TitleSlugDescriptionModel
+from guardian.shortcuts import assign_perm
+
+from grandchallenge.challenges.models import get_logo_path
+from grandchallenge.core.models import UUIDModel
+from grandchallenge.core.validators import JSONSchemaValidator
+from grandchallenge.subdomains.utils import reverse
+
+
+HANGING_LIST_SCHEMA = {
+    "definitions": {},
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "type": "array",
+    "title": "The Hanging List Schema",
+    "items": {
+        "$id": "#/items",
+        "type": "object",
+        "title": "The Items Schema",
+        "required": ["main"],
+        "additionalProperties": False,
+        "properties": {
+            "main": {
+                "$id": "#/items/properties/main",
+                "type": "string",
+                "title": "The Main Schema",
+                "default": "",
+                "examples": ["im1.mhd"],
+                "pattern": "^(.*)$",
```

Comment:
Is this auto-generated? This is not necessary and quite verbose.

Reply:
I think that it came from something that was auto generated. Feel free to change it but check that the JSON editor widget is happy with it (I remember getting problems there when I implemented a basic schema)